extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -262,97 +483,215 @@ The vendor service configuration must reference the service ID and can contain a
+Using Microdata, the markup could look like this:
+
+```html
+<div itemscope itemtype="http://schema.org/NewsArticle">
+  <meta itemprop="isAccessibleForFree" content="false" />
+  <div
+    itemprop="isPartOf"
+    itemscope
+    itemtype="http://schema.org/CreativeWork http://schema.org/Product">
+    <meta itemprop="name" content="The Norcal Tribune" />
+    <meta itemprop="productID" content="norcal_tribute.com:basic" />
+  </div>
+</div>
+```
+
+A usable configuration will provide `NewsArticle` typed item with `isAccessibleForFree` property and a subitem of type `Product` that specifies the `productID`.
+
+In this example:
+
+1.  The product ID is "norcal_tribune.com:basic" (`"productID": "norcal_tribune.com:basic"`).
+2.  This document is currently locked (`"isAccessibleForFree": false`).
+
+The configuration is resolved as soon as `productID` and `isAccessibleForFree` are found. It is, therefore, advised to place the configuration as high up in the DOM tree as possible.
 
-Actions are provided in the "local" service configuration in the "actions" property. It's a named set of action. Any number of actions can be configured this way, but two actions are required: "login" and "subscribe".
 
-All actions work the same way: the popup window is opened for the specified URL. The page opened in the popup window can perform the target action, such as login/subscribe/etc, and it's expected to return by redirecting to the URL specified by the "return" query parameter.
+## Attributes
+### `subscription-action`
+
+An action declared in the "actions" configuration can be marked up using `subscriptions-action` attribute.
 
-Notice, while not explicitly visible, any vendor service can also implement its own actions. Or it can delegate to the "login" service to execute "login" or "subscribe" action.
+Available values:
+- `login`: this will trigger the [Login page][4] of the selected service.
+- `subscribe`: this will trigger the [Subscribe page][5] of the selected service.
 
-### Action delegation
+For instance, this button will execute the "subscribe" action:
+
+```html
+<button subscriptions-action="subscribe" subscriptions-display="EXPR">
+  Subscribe now
+</button>
+```
+
+By default, the actions are hidden and must be explicitly shown using the `subscriptions-display` expression.
+
+### `subscription-service`
 
 In the markup the actions can be delegated to other services for them to execute the actions. This can be achieved by specifying `subscriptions-service` attribute.
 
-e.g. In order to ask google subscriptions to perform subscribe even when `local` service is selected:
+Available values:
+- `local`: this will force the `local` service to be used for a particular action.
+- `{serviceId}` (e.g. `subscribe.google.com`): this will force the service with ID `serviceId` to be used for a particular action.
+
+For instance, this button will the `subscribe.google.com` service to perform the `subscribe` action even when `local` service is selected:
```

Comment:
I don't understand this sentence.

Reply:
Changing to:

> For example, this button will surface the subscribe page from the `subscribe.google.com`  service, regardless of the [service score factors][9]:

Hopefully this makes more sense.

Reply:
Yes, much better!